plugins/inputs/etherscan/README.md

```diff
@@ -0,0 +1,47 @@
+# Etherscan Input Plugin
+
+Gather account balance (ETH) from [Etherscan][] Ethereum block explorer.
+
+**Note:** Telegraf also contains the [webhook][] input which can be used as an
+alternative method for gathering API data.
+
+**Note:** Currently only ETH reporting is available but erc20 support is on the way.
```

Comment:
Could you please remove the hint for upcoming features and only leave in what is available _now_.